docs/concepts/overview/components.md

```diff
@@ -15,9 +15,9 @@ deliver a functioning Kubernetes cluster.
 Master components provide the cluster's control plane. Master components make global decisions about the
 cluster (for example, scheduling), and detecting and responding to cluster events (starting up a new pod when a replication controller's 'replicas' field is unsatisfied).
 
-Master components can be run on any node in the cluster. However,
+Master components can be run on any machine in the cluster. However,
```

Comment:
I agree that VM implies a constrain that doesn't exist and is an excellent change, but I suspect that `machine` implies physical to many people. I think using the more generic reference to `Node` here would be "more correct", as a Node can be either VM or physical machine - and node is very commonly referenced elsewhere in the documentation, where machine isn't.

Reply:
For me as a Kubernetes newbie the term "node" used in two meanings (as a virtual/physical machine and as a Kubernetes "worker node") made a lot of confusion in understanding the architecture concepts. Does it mean that Node (VM, physical machine) and Node (Kubernetes' worker) are the same thing? By the change I made I wanted to separate those two concepts.
What do you thing about using "VM or physical machine" term here -- I've seen it somewhere in the documentation.

Reply:
The two concepts overlap - _worker node_ being a a physical or virtual machine with an OS installed on it that specifically isn’t a _master node_. In each of these cases, node refers to either a physical machine or a virtual machine. The worker vs. master is the idea of when you have a set of machines, you may want to segregate some of them to just have responsibilities to coordinate the others. That’s the the _master nodes_ are in this case, and the _worker nodes_ are all the others in the cluster on which you can run Pods, services, etc.

You bring up a really good point about it being confusing, and we’d like to make that better. WIth the above explanation, do you have a suggestion of where we could improve beyond the scope of just these sentences? I’m sure you’re not the only one getting stuck on this point.

Reply:
I'm wondering why the _Worker Node_ across the documentation is called a _Node_ instead of a _Worker_. From my experience using the common term _Node_ for this specifically dedicated machine made a lot of confusion. 

I raised #7204 for this case, maybe we could continue the discussion there? I would really appreciate your comments.

How do you fell about completing this PR as it is?